### PR TITLE
Add External IDs Support for Group

### PR DESCRIPTION
**Pull request by enhaocui**
This change adds external ids support for Group.

### CODE REVIEW

File: nsxt/resource_nsxt_policy_group.go [hunk 1]
```diff
@@ -117,6 +124,27 @@ func getMACAddressExpressionSchema() *schema.Resource {
...
+				Type:        schema.TypeSet,
+				Required:    true,
+				Description: "List of external ids",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"member_type": {
+				Type:         schema.TypeString,
+				Required:     true,
```
Comment (annakhm) [inline nsxt/resource_nsxt_policy_group.go:140]: I think we can make this optional with default being `VirtualMachine`

Comment (enhaocui) [reply nsxt/resource_nsxt_policy_group.go:140]: Sure. Let me change it. Thanks!

**Comment by annakhm** [comment]
```TestAccResourceNsxtGlobalPolicyGroup_externalIDCriteria
    resource_nsxt_policy_group_test.go:139: Step 1/2 error: Check failed: Check 7/9 error: nsxt_policy_group.test: list or set attribute 'conjunction' must be checked by element count key (conjunction.#) or element value keys (e.g. conjunction.0). Set element value checks should use TestCheckTypeSet functions instead.```

This is a result of recent SDK upgrade..

**Review by annakhm — APPROVED** [review]

**Pull request merged**